src/otx/algorithms/visual_prompting/tasks/inference.py

```diff
@@ -191,29 +186,31 @@ def get_model(config: DictConfig, state_dict: Optional[OrderedDict] = None):
                 "No trained model in project yet. Created new model with '%s'",
                 self.model_name,
             )
-        elif ("path" in otx_model.model_adapters) and (
-            otx_model.model_adapters.get("path").endswith(".ckpt")  # type: ignore[attr-defined]
-        ):
-            # pytorch lightning checkpoint
-            if not otx_model.model_adapters.get("resume"):
-                # If not resuming, just load weights in LightningModule
-                logger.info("Load pytorch lightning checkpoint.")
+        elif otx_model.model_adapters.get("resume", False):
+            # If resuming, pass this part to load checkpoint in Trainer
+            logger.info(f"To resume {otx_model.model_adapters.get('path')}, the checkpoint will be loaded in Trainer.")
+
         else:
-            # pytorch checkpoint saved by otx
+            # Load state_dict
             buffer = io.BytesIO(otx_model.get_data("weights.pth"))
             model_data = torch.load(buffer, map_location=torch.device("cpu"))
-            if model_data.get("model", None) and model_data.get("config", None):
+            if model_data.get("state_dict", None) and model_data.get("pytorch-lightning_version", None):
```

Comment:
It's just a question. why is `pytorch-lightning_version` checked?

Reply:
Just to check if `model_data` is from pytorch lightning checkpoint.
There are many metas such as `epoch`, `state_dict`, `loop`, ...
I chose two of them, `state_dict` and `pytorch-lightning_version`.

Reply:
Thanks for reply. Then could you add simple comment to explain why it's checked?

Reply:
It is just to check if `model_data` has `pytorch-lightning_version`, its value is not important.
Other metas can be checked instead of `pytorch-lightning_version`.

Reply:
Sorry for confusion. I just want to add a simple comment at that code to give explanation to other developers for better understanding.

Reply:
Done.